docs/Kube-API.md

```diff
@@ -132,7 +138,8 @@ You will likely need to adapt those for your own needs.
 * [NMState Config](crds/nmstate.yaml)
 * [Hive PullSecret Secret](crds/pullsecret.yaml)
 * [Hive ClusterDeployment](crds/clusterDeployment.yaml)
```

Comment:
Should be also removed? as SNO there's an agentClusterInstall.yaml

Reply:
ClusterDeployment is still needed in all cases.
It will need to point to a AgentClusterInstall and also the other way around